WTWSMS/history/titles/d_cordoba.txt

```diff
@@ -2,10 +2,13 @@
 	holder=36553 # Opimius Gaius Comenciolus 
 	liege = e_byzantium
 }
-560.1.1={ 
+550.1.1={ 
 	holder=36558 # Flavius Constans    
 	liege = 0
 }
+572.1.1={
+    holder=212873
+}
 586.1.1={ 
```

Comment:
Same as above.

Reply:
It is historically accurate that the rebellion started c.550 in Cordoba and that it wasn't defeated until 572.

Reply:
The map I posted was 575, with the rebellion not defeated. According to the map it should have been in 572, but I don't see the start date?

Reply:
I´m sorry I don´t understand this.

Reply:
Nevermind, as long as they are synchronised between the county and the duchy.